lib/suspenders/adapters/heroku.rb

```diff
@@ -0,0 +1,67 @@
+module Suspenders
+  module Adapters
+    class Heroku
+      def initialize(app_builder)
+        @app_builder = app_builder
+      end
+
+      def set_heroku_remotes
+        remotes = <<-SHELL.strip_heredoc
+
+          # Set up the staging and production apps.
+          #{join_heroku_app('staging')}
```

Comment:
This was taken directly from the `app_builder`, I didn't change any of the method names, but I'm more than willing to change them if this is something you feel should all be done at this time.